hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1631,8 +1634,15 @@ public void checkAcls(ResourceType resType, StoreType storeType,
       ACLType aclType, String vol, String bucket, String key,
       UserGroupInformation ugi, InetAddress remoteAddress, String hostName)
       throws OMException {
-    checkAcls(resType, storeType, aclType, vol, bucket, key,
-        ugi, remoteAddress, hostName, true);
+    String whiteList = configuration.get(OZONE_WRITE_LIST);
+    Set<String> whiteListSet = new HashSet<>();
+    if (whiteList != null) {
+      whiteListSet.addAll(Arrays.asList(whiteList.trim().split(",")));
```

Comment:
Same here and the whitelist should be member OM. We cannot afford reading config every time the stack reaches here.